hw/ip_templates/racl_ctrl/data/racl_ctrl.hjson.tpl

```diff
@@ -32,9 +32,16 @@
   bus_interfaces: [
     { protocol: "tlul", direction: "device", static_racl_support: true }
   ],
-  // In order to not disturb the racl_ctrl address map, we place the alert test
-  // register manually at a safe offset after the main CSRs.
+  // In order to not disturb the racl_ctrl address map, we place the alert test and interrupt
+  // registers manually at a safe offset after the main CSRs.
+  no_auto_intr_regs: "True",
   no_auto_alert_regs: "True",
```

Comment:
I'm a bit confused here. Why are we treating racl_ctrl as a special case for upstream IP here? Preventing the automatic CSR generation while actually still having the CSRs means we can't do the automatic tests.

Reply:
Perhaps we need some better hjson descriptions to describe that the interrupt CSRs are actually present, just at a different offset?